merkledag/README.md

```diff
@@ -0,0 +1,144 @@
+# The merkledag
+
+
+Authors: [Juan Benet](github.com/jbenet)
+
+Reviewers:
+
+- [Jeromy Johnson](github.com/whyrusleeping)
+
+* * *
+
+This is the [Spec](../) for the IPFS Merkledag.
+
+## Definitions
+
+- `hash` - throughout this document, the word `hash` refers specifically to cryptographic hash functions, such as sha3.
+- `dag` - directed acyclic graph
+- `merkle-link` - a link (graph edge) between two objects, which is (a) represented by the hash of the target object, and (b) embedded in the source object. merkle-links construct graphs (dags) whose links are content-addressed, and authenticated.
+- `merkledag` - the merkledag is a directed acyclic graph whose links are merkle-links (hashes of the content). It is a hash tree, and (under a very lose definition) a merkle tree. Alternative names: the merkle-web, the merkle-forest, the merkle-chain.
+- `multihash` - the [multihash](https://github.com/jbenet/multihash) format / protocol.
+- `ipfs object` - a node in the ipfs merkledag. It represents a singular entity.
```

Comment:
On Fri, May 29, 2015 at 11:34:14AM -0700, Juan Batiz-Benet wrote:
> > +- `ipfs object` - a node in the ipfs merkledag. It represents a
> >    singular entity.
> 
> An independent entity. 

So “indepenently addressable”?  Or fetchable?  Hashable?  Verifiable?
All of the above?